.github/actions/conda_build_publish_package/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Andrew Prokhorenkov
```

Comment:
Fix names and year

Reply:
I will remove the LICENSE file of the action since it is a local directory and no longer a separate project (I copied the files from my fork of the original action and took the LICENSE with me).